### PR TITLE
Escape HTML on display and publication of title and description

### PR DESCRIPTION
**Pull request by jeffsikes**
Prior to display on Postmarks or publishing to the ActivityPub outbox, escape HTML characters in the user submitted title and description. Future enhancements will allow for markdown.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,4 +54,4 @@
     "nodemon": "^2.0.20",
     "prettier": "^3.0.3"
   }
-}
+}
```
Comment (ckolderup) [inline package.json:57]: can you add a final linebreak back to this file? I guess eslint doesn't check the package.json-- maybe that's possible in the future with some better configuration

File: .DS_Store
Comment (ckolderup) [inline .DS_Store:1]: please delete this file-- maybe it should be in the .gitignore, but it's some macOS metadata that we don't want to clutter up the repo.

**Review by ckolderup — CHANGES_REQUESTED** [review]
almost done, just a few small changes!

File: .DS_Store
Comment (jeffsikes) [reply .DS_Store:1]: Deleted and added to .gitignore as well - pesky!

**Review by ckolderup — APPROVED** [review]
Thank you Jeff! I really appreciate the help.

File: package.json (hunk 1 above)
Comment (jeffsikes) [reply package.json]: Final linebreak added. Sorry about the ES Lint multi commits. I'm running it locally now before a push.

**Pull request merged**